core/admin/mailu/internal/nginx.py

```diff
@@ -27,23 +27,26 @@
 
 WEBMAIL_PORTS = ['10143', '10025']
 
-def check_credentials(user, password, ip, protocol=None, auth_port=None):
+def check_credentials(user, password, ip, protocol=None, auth_port=None, source_port=None):
     if not user or not user.enabled or (protocol == "imap" and not user.enable_imap and not auth_port in WEBMAIL_PORTS) or (protocol == "pop3" and not user.enable_pop):
+        app.logger.info(f'Login attempt for: {user}/{protocol}/{auth_port} from: {ip}/{source_port}: failed: accound disabled')
```

Comment:
Hmm, splitting the port with "/" is strange...
It should be "account disabled"

Reply:
The idea was to be able to ``grep|cut -d: |cut -d/ `` easily ... What would you rather use? ``!``? ``|``?